### PR TITLE
Basic api information and versioning

### PR DESCRIPTION
**Pull request by bahrmichael**

### CODE REVIEW

**Comment by bahrmichael** [comment]
* Added the link to an intro video in the description as well as the endpoint URLs.
* Added info that solar systems statistics are not yet third party enabled (but might be released soon).
* Replaced POST with call, as an unauthenticated POST will not give any results, but a regular GET will. This is to avoid confusion on the reader's side.
* Updated the versions information to the current (and only available) api-v5.
* Added on actually available versions.
* Example for the X-Deprecated Header.

File: docs/crest/intro.md [hunk 1]
```diff
@@ -5,10 +5,15 @@ CREST is the new RESTfull API, destined to replace the older XML API. Currently
...
+There is a good [video](https://www.youtube.com/watch?v=QMQOjUjrZIo) by CCP Zero Gravitas explaining the CREST API and the its basic concepts.
+
 ## Walking
 CREST is designed to be a properly RESTful API with good support for versioning and changes. As such, a well designed app will not break when things are changed on the API (most of the time!). To enable this one of the things your app should do is generate links to endpoints dynamically. The only hard coded URL will be to the root resource. Here you will find a list of all the other top level endpoints, with href elements for each.
 
-To get the list of endpoints, you simply need to make an HTTP GET request to the appropriate root (SISI or TQ).
+To get the list of endpoints, you simply need to make an HTTP GET request to the appropriate root:
+
+* TQ: `https://crest-tq.eveonline.com`
+* SISI: `https://api-sisi.testeveonline.com/`
```
Comment (CarbonAlabel) [inline docs/crest/intro.md]: TQ and SISI aren't the only two servers. Also, root URLs are already listed on the [Quick Reference](https://eveonline-third-party-documentation.readthedocs.io/en/latest/reference/reference.html) page so simply linking to that page would be a better choice here.

**Comment by bahrmichael** [comment]
Added map_moons and changed the endpoint information to point to the reference. Can't test if the latter's URL is correct due to some locale bug on my machine :/

**Comment by CarbonAlabel** [comment]
Appears to be working.

**Comment by polarina** [comment]
The commits in this pull request need to be cleaned up before we can merge them, sadly. :(

Can you split fc08a26 into separate commits per logical change, please?
The commit message for 09feb4a29734a7558a5a40cee6c66dab81799102 should be changed, as it is not reflecting the change made.
The commits 4d2be3e5c1c36d68441eeeff33f79917d950a3c3 and 95b41c6d26764a6cc943ddcccc9312970931ebae are identical.
The commits 42d2670, 2fc1c33, f0b61c6, eb31395, and parts of ffe7a90 and ab14dce  should merged into a single commit.

**Comment by bahrmichael** [comment]
Will maybe add that information in a later PR again. No time right now.

**Pull request closed without merging**